### PR TITLE
#157774471  Adds Admin sign in page

### PR DESCRIPTION
**Pull request by artorious**
#### What does this PR do?
Adds navigation to the web app's administrator sign-in page

#### Description of Task to be completed?
A user should be able to navigate to the web app's homepage, and from there, the administrator sign-in page

On Webbrowser:
* Download the [Zip from GitHub](https://github.com/artorious/MaintenanceTracker/archive/ft-admin-signin-%23157774471.zip)
* Extract the MaintenanceTracker folder.
* In the IU folder,  open  `index.html` on a browser

On a  terminal (Linux):
* Install and set up git `sudo apt install git`
* Clone the repository `git clone https://github.com/artorious/MaintenanceTracker.git`
* Open  `MantenanceTracker/UI/index.html` on a browser
* Navigate to admin sign in by clicking `admin` on the navigation bar

#### Any background context you want to provide?
The Website UI is to be developed in HTML5 & CSS3 

#### What are the relevant pivotal tracker stories?
[#157774471](https://www.pivotaltracker.com/story/show/157774471)

#### Screenshots 
admin_signin_screenshot

#### Questions:

### CODE REVIEW

**Comment by johnmusiu** [comment]
PR title looks a little different from the Andela guide provided on the same. Check it out [here](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)

The rest of the PR looks ok.
Good responsive UI design.

**Comment by artorious** [comment]
@johnmusiu ,
Good catch.
I've now realized, I used the branch naming convention.  will see about changing that.

 [On slack you pointed out the screenshots,](https://open-andela.slack.com/archives/C7N7DN2CQ/p1527228518000167?thread_ts=1527223589.000208&cid=C7N7DN2CQ) . My homepage has a sign-up form as well.

Will make a few tweaks to make the distinction

File: UI/admin_signin.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <a href="#">
+        <img src="images/user_avatar_sm-46px.png" alt="profile picture">
+      </a>
+    </nav>
+  </header>
+  <hr>
+  <main>
+    <div>
+      <h2>Notice Board</h2>
+      <p>
```
Comment (artorious) [inline UI/admin_signin.html]: Maybe add `<h4> Attn Admins<h4>`?

**Review by artorious — COMMENTED** [review]
Make changes

File: UI/admin_signin.html (hunk 1 above)
Comment (johnmusiu) [reply UI/admin_signin.html]: Try that, see if it makes a good difference

Also for the PR title remove the [ ] around the PT story ID

**Pull request merged**